.github/workflows/asset-lint.yml

```diff
@@ -0,0 +1,80 @@
+name: Lint Scripts, Styles, and Texts
+
+on:
+  workflow_call:
+    secrets:
+      CHECKOUT_TOKEN:
+        required: false
+        description: >
+          The GitHub token to authenticate checkout with. Pass in a GitHub personal access token if authenticated
+          submodules are used.
+
+    inputs:
+      cancel-workflow-on-failure:
+        description: When set to "true", it will cancel the current workflow run with all jobs if this workflow fails.
+        type: string
+        default: 'true'
+      scripts:
+        description: |
+          Information about the projects where JavaScript linting should occur. Can be in two formats:
+          - A comma separated list of relative paths to the project directories. These are the paths where a package.json
+            file could be, if desired. If there is no package.json on the provided path, the default one from this
+            repository is copied and the default asset subpath (./Assets/Scripts) is assumed. Using this approach, you have
+            to copy the assets to their final path (in ./wwwroot) using a different method, for example using the Copy
+            task in MSBuild.
+          - JSON object where the property names are the relative paths to the project directory where the package.json
+            file would be. For each path, a new package.json file is created from the template in this repository and then
+            the value of the property is placed into the $.nodejsExtensions path in the project file.
+        type: string
+        default: .:.
```

Comment:
Let's use a simple globbing config here as well, like for CSS. Isn't that suitable?